scripts/commands/setrank.lua

```diff
@@ -0,0 +1,38 @@
+---------------------------------------------------------------------------------------------------
+-- func: setrank
+-- desc: Sets the players rank.
+---------------------------------------------------------------------------------------------------
+
+cmdprops =
+{
+    permission = 1,
+    parameters = "si"
+}
+
+function error(player, msg)
+    player:PrintToPlayer(msg)
+    player:PrintToPlayer("!setrank <player> <new rank>")
+end
+
+function onTrigger(player, target, rank)
+    local targ
```

Comment:
```lua
    if target == nil then
        error(player, "Set who's rank?")
        return
    end

    local targ = GetPlayerByName(target)
```
should be fine since targ isn't used before this line

Reply:
sorry for having a small nitpicky looking thing every time you've made a change, just been browsing between support calls tonight. I think this is the last one.

Reply:
kk